.storybook/main.js

```diff
@@ -11,17 +11,11 @@ const {
 const tsConfigPath = path.resolve(__dirname, '../tsconfig.base.json');
 
 /**
- * @typedef {import('@storybook/core-common').StorybookConfig} StorybookBaseConfig
+ * @typedef {import('./types').StorybookConfig} StorybookBaseConfig
```

Comment:
re-declaring is necessary in order to not have to refactor all storybook configs within repo and to expose these type alias within javascript context so all other packages importing from `main.js` will still get access to these types.